AI/BattleAI/AttackPossibility.cpp

```diff
@@ -175,7 +175,7 @@ AttackPossibility AttackPossibility::evaluate(const BattleAttackInfo & attackInf
 				if (!attackInfo.shooting && defenderState->ableToRetaliate() && !counterAttacksBlocked)
 				{
 					damageReceived = averageDmg(retaliation);
-					attackerDamageReduce = calculateDamageReduce(defender, attacker, damageReceived, state);
```

Comment:
Why do you think it is a bug?

Reply:
Retalitation is when defender attacks attacker

Reply:
But this is code is for *retaliation*, so defender and attacker are now swapped.

Wrong, no changes needed here